code/modules/awaymissions/corpse.dm

```diff
@@ -21,6 +21,10 @@
 	density = 1
 	anchored = 1
 
+/obj/effect/mob_spawn/New()
+	if(!mob_gender)
+		mob_gender = pick(MALE, FEMALE)
+
```

Comment:
The idea is that it's a new character, not a continuation of the old one. Ghost prefs have no sway here.